### ISSUE TITLE
Error when adding new show

### ISSUE THREAD

**Issue by manrata** [SickChill/SickChill#5692]
Tried adding several different shows, but the error is the same each time.

Tried reinstalling sickchill, keeping settings, still same issue. 
Sickchill still finds shows already added.
No files appear in new show folder at all.

Branch/Commit: Master/Commit: e094759640ca61c5c033bf1ff9d33bc7fc1ba857
OS: Windows 10
Browser: Chrome/Firefox
What you did: Tried adding new show
What happened: After finding show, and clicking add show, could see on the show list it was loading, but then nothing, and an error in the log
What you expected: The show to be added
Logs:
```
2019-11-17 01:21:16 WARNING ThreadPoolExecutor-0_1 :: Show id 368611 not found on theTVDB, not adding the show: error "There is no item named u'en.xml' in the archive" 2019-11-17 01:21:16 INFO ThreadPoolExecutor-0_1 :: theTVDB: {u'useZip': True, u'apikey': u'F9C450E78D99172E', u'cache': u'C:\\SickChill\\Data\\cache\\indexers\\theTVDB', u'language': u'en'} AA tvdb_error: "There is no item named 'en.xml' in the archive" AA raise tvdb_error(e) AA File "C:\SickChill\SickChill\lib\tvdb_api\tvdb_api.py", line 629, in _getetsrc AA epsEt = self._getetsrc(url, language=language) AA File "C:\SickChill\SickChill\lib\tvdb_api\tvdb_api.py", line 874, in _getShowData AA self._getShowData(key, self.config['language'], True) AA File "C:\SickChill\SickChill\lib\tvdb_api\tvdb_api.py", line 946, in __getitem__ AA s = t[self.indexer_id] AA File "C:\SickChill\SickChill\sickbeard\show_queue.py", line 324, in run AA Traceback (most recent call last): 2019-11-17 01:18:19 ERROR SHOWQUEUE-ADD :: [e094759] Unable to look up the show in C:\SickChill\The Mandalorian on theTVDB using ID 361753, not using the NFO. Delete .nfo and try adding manually again.: "There is no item named 'en.xml' in the archive" 2019-11-17 01:18:18 INFO SHOWQUEUE-ADD :: theTVDB: {u'useZip': True, u'apikey': u'F9C450E78D99172E', u'cache': u'C:\\SickChill\\Data\\cache\\indexers\\theTVDB', u'language': 'en'} 2019-11-17 01:18:18 INFO SHOWQUEUE-ADD :: Starting to add show by ShowDir: C:\SickChill\The Mandalorian 2019-11-17 00:54:37 INFO SEARCHQUEUE-DAILY-SEARCH :: Error connecting to sab, no data returned 2019-11-17 00:54:37 INFO SEARCHQUEUE-DAILY-SEARCH :: Request failed: HTTPConnectionPool(host='192.168.1.100', port=8080): Max retries exceeded with url: /api?apikey=**********&name=https%3A%2F%2Fapi.nzbgeek.info%2Fapi%3Ft%3Dget%26id%3D2b7d4a7f27efec000f244b91894d25e9%26apikey%3D9b4f90a886f84ec6c129951fa2ee8411&cat=tv&priority=1&mode=addurl&output=json (Caused by NewConnectionError(': Failed to establish a new connection: [Errno 10061] No connection could be made because the target machine actively refused it',)) () 2019-11-17 00:54:36 INFO SEARCHQUEUE-DAILY-SEARCH :: Sending NZB to SABnzbd 2019-11-17 00:54:36 INFO SEARCHQUEUE-DAILY-SEARCH :: Downloading See.S01E05.720p.ATVP.WEB-DL.DD5.1.H.264-CasStudio-Obfuscated from NZBGeek 2019-11-17 00:54:36 INFO SEARCHQUEUE-DAILY-SEARCH :: [NZBGeek] :: Quality of See.S01E05.Plastic.720p.WEB-DL.DD5.1.H.264-eztv is 720p WEB-DL 2019-11-17 00:54:36 INFO SEARCHQUEUE-DAILY-SEARCH :: [NZBGeek] :: Quality of See.S01E05.720p.ATVP.WEB-DL.DD5.1.H.264-CasStudio-Obfuscated is 720p WEB-DL 2019-11-17 00:54:36 INFO SEARCHQUEUE-DAILY-SEARCH :: [NZBGeek] :: Found result See.S01E05.Plastic.720p.WEB-DL.DD5.1.H.264-eztv at https://api.nzbgeek.info/api?t=get&id=b84119faaa08eab42a94ee562e02469c&apikey=********** 2019-11-17 00:54:36 INFO SEARCHQUEUE-DAILY-SEARCH :: [NZBGeek] :: Found result See.S01E05.720p.ATVP.WEB-DL.DD5.1.H.264-CasStudio-Obfuscated at https://api.nzbgeek.info/api?t=get&id=2b7d4a7f27efec000f244b91894d25e9&apikey=********** 2019-11-17 00:54:35 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-17 00:54:32 INFO FINDSUBTITLES :: No subtitles to download 2019-11-17 00:54:32 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-17 00:54:32 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-17 00:54:32 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-17 00:47:46 INFO ThreadPoolExecutor-0_2 :: No update needed 2019-11-17 00:47:45 INFO ThreadPoolExecutor-0_2 :: Checking for updates using GIT AA tvdb_error: "There is no item named 'en.xml' in the archive" AA raise tvdb_error(e) AA File "C:\SickChill\SickChill\lib\tvdb_api\tvdb_api.py", line 629, in _getetsrc AA epsEt = self._getetsrc(url, language=language) AA File "C:\SickChill\SickChill\lib\tvdb_api\tvdb_api.py", line 874, in _getShowData AA self._getShowData(key, self.config['language'], True) AA File "C:\SickChill\SickChill\lib\tvdb_api\tvdb_api.py", line 946, in __getitem__ AA s = t[self.indexer_id] AA File "C:\SickChill\SickChill\sickbeard\show_queue.py", line 324, in run AA Traceback (most recent call last): 2019-11-17 00:45:24 ERROR SHOWQUEUE-ADD :: [e094759] Unable to look up the show in C:\SickChill\The Mandalorian on theTVDB using ID 361753, not using the NFO. Delete .nfo and try adding manually again.: "There is no item named 'en.xml' in the archive" 2019-11-17 00:45:24 INFO SHOWQUEUE-ADD :: theTVDB: {u'useZip': True, u'apikey': u'F9C450E78D99172E', u'cache': u'C:\\SickChill\\Data\\cache\\indexers\\theTVDB', u'language': 'en'} 2019-11-17 00:45:24 INFO SHOWQUEUE-ADD :: Starting to add show by ShowDir: C:\SickChill\The Mandalorian AA None 2019-11-17 00:44:57 ERROR ThreadPoolExecutor-0_1 :: [e094759] Missing time zone for network: Apple TV+. Check valid network is set in indexer (theTVDB) before filing issue. AA None 2019-11-17 00:44:57 ERROR ThreadPoolExecutor-0_1 :: [e094759] Missing time zone for network: Peacock. Check valid network is set in indexer (theTVDB) before filing issue. 2019-11-17 00:44:33 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-17 00:44:33 INFO TORNADO :: Starting SickChill on http://0.0.0.0:8081/ 2019-11-17 00:44:33 INFO MAIN :: Forcing web server to port 8081 2019-11-17 00:44:33 INFO MAIN :: Starting SickChill [master] using 'C:\SickChill\Data\config.ini' 2019-11-17 00:44:33 INFO MAIN :: 368747: Unable to find IMDb show info in the database 2019-11-17 00:44:32 INFO MAIN :: Successfully downloaded unrar.exe and set as unrar tool 2019-11-17 00:44:31 INFO MAIN :: Trying to download unrar.exe and set the path 2019-11-17 00:41:22 INFO EVENT-QUEUE :: Waiting for the CHECKVERSION thread to exit 2019-11-17 00:41:22 INFO EVENT-QUEUE :: Waiting for the SHOWUPDATER thread to exit 2019-11-17 00:41:22 INFO EVENT-QUEUE :: Waiting for the BACKLOG thread to exit 2019-11-17 00:41:21 INFO EVENT-QUEUE :: Waiting for the DAILYSEARCHER thread to exit 2019-11-17 00:41:21 INFO EVENT-QUEUE :: Aborting all threads 2019-11-17 00:41:21 INFO MAIN :: Signal 2 caught, saving and exiting... 2019-11-17 00:41:11 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-17 00:41:11 INFO TORNADO :: Starting SickChill on http://0.0.0.0:8081/ 2019-11-17 00:41:11 INFO MAIN :: Forcing web server to port 8081 2019-11-17 00:41:11 INFO MAIN :: Starting SickChill [master] using 'C:\SickChill\Data\config.ini' 2019-11-17 00:41:11 INFO MAIN :: 368747: Unable to find IMDb show info in the database 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Shutting down Tornado 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Saving config file to disk 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Saving all shows to the database 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Waiting for the EVENT-QUEUE thread to exit 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Waiting for the FINDSUBTITLES thread to exit 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Waiting for the FINDPROPERS thread to exit 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Waiting for the TRAKTCHECKER thread to exit 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Waiting for the POSTPROCESSOR thread to exit 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Waiting for the POSTPROCESSOR thread to exit 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Waiting for the SEARCHQUEUE-DAILY-SEARCH thread to exit 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Waiting for the SHOWQUEUE-ADD thread to exit 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Waiting for the CHECKVERSION thread to exit 2019-11-17 00:40:58 INFO EVENT-QUEUE :: Waiting for the SHOWUPDATER thread to exit 2019-11-17 00:40:57 INFO EVENT-QUEUE :: Waiting for the BACKLOG thread to exit 2019-11-17 00:40:57 INFO EVENT-QUEUE :: Waiting for the DAILYSEARCHER thread to exit 2019-11-17 00:40:57 INFO EVENT-QUEUE :: Aborting all threads 2019-11-17 00:39:26 INFO SEARCHQUEUE-DAILY-SEARCH :: Error connecting to sab, no data returned 2019-11-17 00:39:26 INFO SEARCHQUEUE-DAILY-SEARCH :: Request failed: HTTPConnectionPool(host='192.168.1.100', port=8080): Max retries exceeded with url: /api?apikey=**********&name=https%3A%2F%2Fapi.nzbgeek.info%2Fapi%3Ft%3Dget%26id%3D2b7d4a7f27efec000f244b91894d25e9%26apikey%3D9b4f90a886f84ec6c129951fa2ee8411&cat=tv&priority=1&mode=addurl&output=json (Caused by NewConnectionError(': Failed to establish a new connection: [Errno 10061] No connection could be made because the target machine actively refused it',)) () 2019-11-17 00:39:24 INFO SEARCHQUEUE-DAILY-SEARCH :: Sending NZB to SABnzbd 2019-11-17 00:39:24 INFO SEARCHQUEUE-DAILY-SEARCH :: Downloading See.S01E05.720p.ATVP.WEB-DL.DD5.1.H.264-CasStudio-Obfuscated from NZBGeek 2019-11-17 00:39:24 INFO SEARCHQUEUE-DAILY-SEARCH :: [NZBGeek] :: Quality of See.S01E05.Plastic.720p.WEB-DL.DD5.1.H.264-eztv is 720p WEB-DL 2019-11-17 00:39:24 INFO SEARCHQUEUE-DAILY-SEARCH :: [NZBGeek] :: Quality of See.S01E05.720p.ATVP.WEB-DL.DD5.1.H.264-CasStudio-Obfuscated is 720p WEB-DL 2019-11-17 00:39:24 INFO SEARCHQUEUE-DAILY-SEARCH :: [NZBGeek] :: Found result See.S01E05.Plastic.720p.WEB-DL.DD5.1.H.264-eztv at https://api.nzbgeek.info/api?t=get&id=b84119faaa08eab42a94ee562e02469c&apikey=********** 2019-11-17 00:39:24 INFO SEARCHQUEUE-DAILY-SEARCH :: [NZBGeek] :: Found result See.S01E05.720p.ATVP.WEB-DL.DD5.1.H.264-CasStudio-Obfuscated at https://api.nzbgeek.info/api?t=get&id=2b7d4a7f27efec000f244b91894d25e9&apikey=********** 2019-11-17 00:39:24 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-17 00:39:21 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-17 00:39:21 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-17 00:39:16 INFO FINDSUBTITLES :: No subtitles to download 2019-11-17 00:39:16 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-17 00:35:29 INFO SEARCHQUEUE-BACKLOG-371446 :: No needed episodes found during backlog search for: [Heirs of the Night] 2019-11-17 00:35:27 INFO SEARCHQUEUE-BACKLOG-371446 :: [NZBGeek] :: Performing season pack search for Heirs of the Night 2019-11-17 00:35:04 INFO SEARCHQUEUE-BACKLOG-371446 :: [NZBGeek] :: Performing episode search for Heirs of the Night 2019-11-17 00:35:04 INFO SEARCHQUEUE-BACKLOG-371446 :: Beginning backlog search for: [Heirs of the Night] 2019-11-17 00:35:00 INFO SEARCHQUEUE-BACKLOG-361565 :: Error connecting to sab, no data returned 2019-11-17 00:35:00 INFO SEARCHQUEUE-BACKLOG-361565 :: Request failed: HTTPConnectionPool(host='192.168.1.100', port=8080): Max retries exceeded with url: /api?apikey=**********&name=https%3A%2F%2Fapi.nzbgeek.info%2Fapi%3Ft%3Dget%26id%3D2b7d4a7f27efec000f244b91894d25e9%26apikey%3D9b4f90a886f84ec6c129951fa2ee8411&cat=tv&priority=1&mode=addurl&output=json (Caused by NewConnectionError(': Failed to establish a new connection: [Errno 10061] No connection could be made because the target machine actively refused it',)) () 2019-11-17 00:34:59 INFO SEARCHQUEUE-BACKLOG-361565 :: Sending NZB to SABnzbd 2019-11-17 00:34:59 INFO SEARCHQUEUE-BACKLOG-361565 :: Downloading See.S01E05.720p.ATVP.WEB-DL.DD5.1.H.264-CasStudio-Obfuscated from NZBGeek 2019-11-17 00:34:59 INFO SEARCHQUEUE-BACKLOG-361565 :: [NZBGeek] :: Quality of See.S01E05.Plastic.720p.WEB-DL.DD5.1.H.264-eztv is 720p WEB-DL 2019-11-17 00:34:59 INFO SEARCHQUEUE-BACKLOG-361565 :: [NZBGeek] :: Quality of See.S01E05.720p.ATVP.WEB-DL.DD5.1.H.264-CasStudio-Obfuscated is 720p WEB-DL 2019-11-17 00:34:59 INFO SEARCHQUEUE-BACKLOG-361565 :: [NZBGeek] :: Found result See.S01E05.Plastic.720p.WEB-DL.DD5.1.H.264-eztv at https://api.nzbgeek.info/api?t=get&id=b84119faaa08eab42a94ee562e02469c&apikey=********** 2019-11-17 00:34:59 INFO SEARCHQUEUE-BACKLOG-361565 :: [NZBGeek] :: Found result See.S01E05.720p.ATVP.WEB-DL.DD5.1.H.264-CasStudio-Obfuscated at https://api.nzbgeek.info/api?t=get&id=2b7d4a7f27efec000f244b91894d25e9&apikey=********** 2019-11-17 00:34:59 INFO SEARCHQUEUE-BACKLOG-361565 :: [NZBGeek] :: Performing episode search for See 2019-11-17 00:34:59 INFO SEARCHQUEUE-BACKLOG-361565 :: Beginning backlog search for: [See] 2019-11-17 00:34:55 INFO SEARCHQUEUE-BACKLOG-281534 :: No needed episodes found during backlog search for: [Killjoys] 2019-11-17 00:34:52 INFO SEARCHQUEUE-BACKLOG-281534 :: [NZBGeek] :: Performing season pack search for Killjoys 2019-11-17 00:34:19 INFO SEARCHQUEUE-BACKLOG-281534 :: [NZBGeek] :: Performing episode search for Killjoys 2019-11-17 00:34:19 INFO SEARCHQUEUE-BACKLOG-281534 :: Beginning backlog search for: [Killjoys] 2019-11-17 00:34:17 INFO SEARCHQUEUE-BACKLOG-320766 :: No needed episodes found during backlog search for: [The Good Fight] 2019-11-17 00:34:14 INFO SEARCHQUEUE-BACKLOG-320766 :: [NZBGeek] :: Performing season pack search for The Good Fight 2019-11-17 00:33:39 INFO SEARCHQUEUE-BACKLOG-320766 :: [NZBGeek] :: Performing episode search for The Good Fight 2019-11-17 00:33:39 INFO SEARCHQUEUE-BACKLOG-320766 :: Beginning backlog search for: [The Good Fight] 2019-11-17 00:33:37 INFO SEARCHQUEUE-BACKLOG-317169 :: No needed episodes found during backlog search for: [Zapped!] 2019-11-17 00:33:34 INFO SEARCHQUEUE-BACKLOG-317169 :: [NZBGeek] :: Performing season pack search for Zapped! 2019-11-17 00:33:24 INFO SEARCHQUEUE-BACKLOG-317169 :: [NZBGeek] :: Performing episode search for Zapped! 2019-11-17 00:33:24 INFO SEARCHQUEUE-BACKLOG-317169 :: Beginning backlog search for: [Zapped!] 2019-11-17 00:33:23 INFO SEARCHQUEUE-BACKLOG-317169 :: No needed episodes found during backlog search for: [Zapped!] 2019-11-17 00:33:20 INFO SEARCHQUEUE-BACKLOG-317169 :: [NZBGeek] :: Performing season pack search for Zapped! 2019-11-17 00:33:04 INFO SEARCHQUEUE-BACKLOG-317169 :: [NZBGeek] :: Performing episode search for Zapped! 2019-11-17 00:33:04 INFO SEARCHQUEUE-BACKLOG-317169 :: Beginning backlog search for: [Zapped!] 2019-11-17 00:33:01 INFO SEARCHQUEUE-BACKLOG-317169 :: No needed episodes found during backlog search for: [Zapped!] 2019-11-17 00:32:58 INFO SEARCHQUEUE-BACKLOG-317169 :: [NZBGeek] :: Performing season pack search for Zapped! 2019-11-17 00:32:49 INFO SEARCHQUEUE-BACKLOG-317169 :: [NZBGeek] :: Performing episode search for Zapped! 2019-11-17 00:32:49 INFO SEARCHQUEUE-BACKLOG-317169 :: Beginning backlog search for: [Zapped!] 2019-11-17 00:32:44 INFO SEARCHQUEUE-BACKLOG-363642 :: No needed episodes found during backlog search for: [Sliced (2019)] 2019-11-17 00:32:42 INFO SEARCHQUEUE-BACKLOG-363642 :: [NZBGeek] :: Performing season pack search for Sliced (2019) 2019-11-17 00:32:34 INFO SEARCHQUEUE-BACKLOG-363642 :: [NZBGeek] :: Performing episode search for Sliced (2019) 2019-11-17 00:32:34 INFO SEARCHQUEUE-BACKLOG-363642 :: Beginning backlog search for: [Sliced (2019)] 2019-11-17 00:32:30 INFO SEARCHQUEUE-BACKLOG-300147 :: No needed episodes found during backlog search for: [SuperMansion] 2019-11-17 00:32:27 INFO SEARCHQUEUE-BACKLOG-300147 :: [NZBGeek] :: Performing season pack search for SuperMansion 2019-11-17 00:32:14 INFO SEARCHQUEUE-BACKLOG-300147 :: [NZBGeek] :: Performing episode search for SuperMansion 2019-11-17 00:32:14 INFO SEARCHQUEUE-BACKLOG-300147 :: Beginning backlog search for: [SuperMansion] 2019-11-17 00:32:12 INFO SEARCHQUEUE-BACKLOG-341663 :: No needed episodes found during backlog search for: [Titans (2018)] 2019-11-17 00:32:12 INFO SEARCHQUEUE-BACKLOG-341663 :: [NZBGeek] :: Performing season pack search for Titans (2018) 2019-11-17 00:32:09 INFO SEARCHQUEUE-BACKLOG-341663 :: [NZBGeek] :: Performing episode search for Titans (2018) 2019-11-17 00:32:09 INFO SEARCHQUEUE-BACKLOG-341663 :: Beginning backlog search for: [Titans (2018)] 2019-11-17 00:32:08 INFO SEARCHQUEUE-BACKLOG-77666 :: No needed episodes found during backlog search for: [The Amazing Race (US)] 2019-11-17 00:32:08 INFO SEARCHQUEUE-BACKLOG-77666 :: [NZBGeek] :: Performing season pack search for The Amazing Race (US) 2019-11-17 00:32:04 INFO SEARCHQUEUE-BACKLOG-77666 :: [NZBGeek] :: Performing episode search for The Amazing Race (US) 2019-11-17 00:32:04 INFO SEARCHQUEUE-BACKLOG-77666 :: Beginning backlog search for: [The Amazing Race (US)] 2019-11-17 00:32:03 INFO SEARCHQUEUE-BACKLOG-76733 :: No needed episodes found during backlog search for: [Survivor] 2019-11-17 00:32:03 INFO SEARCHQUEUE-BACKLOG-76733 :: [NZBGeek] :: Performing season pack search for Survivor 2019-11-17 00:31:59 INFO SEARCHQUEUE-BACKLOG-76733 :: [NZBGeek] :: Performing episode search for Survivor 2019-11-17 00:31:59 INFO SEARCHQUEUE-BACKLOG-76733 :: Beginning backlog search for: [Survivor] 2019-11-17 00:31:54 INFO SEARCHQUEUE-BACKLOG-275274 :: No needed episodes found during backlog search for: [Rick and Morty] 2019-11-17 00:31:51 INFO SEARCHQUEUE-BACKLOG-275274 :: [NZBGeek] :: Performing season pack search for Rick and Morty 2019-11-17 00:31:44 INFO SEARCHQUEUE-BACKLOG-275274 :: [NZBGeek] :: Performing episode search for Rick and Morty 2019-11-17 00:31:44 INFO SEARCHQUEUE-BACKLOG-275274 :: Beginning backlog search for: [Rick and Morty] 2019-11-17 00:31:43 INFO SEARCHQUEUE-BACKLOG-110381 :: No needed episodes found during backlog search for: [Archer (2009)] 2019-11-17 00:31:43 INFO SEARCHQUEUE-BACKLOG-110381 :: [NZBGeek] :: Quality of Archer.2009.S10.720p.AMZN.WEB-DL.DDP5.1.H.264-NTb-Obfuscated is 720p WEB-DL 2019-11-17 00:31:38 INFO SEARCHQUEUE-BACKLOG-110381 :: [NZBGeek] :: Performing season pack search for Archer (2009) 2019-11-17 00:31:24 INFO SEARCHQUEUE-BACKLOG-110381 :: [NZBGeek] :: Performing episode search for Archer (2009) 2019-11-17 00:31:24 INFO SEARCHQUEUE-BACKLOG-110381 :: Beginning backlog search for: [Archer (2009)] 2019-11-17 00:31:24 INFO SEARCHQUEUE-BACKLOG-73141 :: No needed episodes found during backlog search for: [American Dad!] 2019-11-17 00:31:19 INFO SEARCHQUEUE-BACKLOG-73141 :: [NZBGeek] :: Performing season pack search for American Dad! 2019-11-17 00:30:47 INFO SEARCHQUEUE-BACKLOG-73141 :: [NZBGeek] :: Performing episode search for American Dad! 2019-11-17 00:30:39 INFO SEARCHQUEUE-BACKLOG-73141 :: Beginning backlog search for: [American Dad!] 2019-11-17 00:30:38 INFO SEARCHQUEUE-BACKLOG-73141 :: No needed episodes found during backlog search for: [American Dad!] 2019-11-17 00:30:34 INFO SEARCHQUEUE-BACKLOG-73141 :: [NZBGeek] :: Performing season pack search for American Dad! 2019-11-17 00:29:19 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-17 00:29:19 INFO SEARCHQUEUE-BACKLOG-73141 :: [NZBGeek] :: Performing episode search for American Dad! 2019-11-17 00:29:19 INFO SEARCHQUEUE-BACKLOG-73141 :: Beginning backlog search for: [American Dad!] 2019-11-17 00:29:15 INFO FINDPROPERS :: Completed the search for new propers, next check in approx. 3h, 59m 2019-11-17 00:29:15 INFO FINDPROPERS :: [OzNzb] :: Searching for any new PROPER releases from OzNzb 2019-11-17 00:29:15 INFO FINDPROPERS :: [NZBGeek] :: Searching for any new PROPER releases from NZBGeek 2019-11-17 00:29:15 INFO FINDPROPERS :: Beginning the search for new propers 2019-11-16 23:59:31 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 23:59:23 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 23:59:21 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 23:59:21 INFO DAILYSEARCHER :: Searching for new released episodes ... AA tvdb_error: "There is no item named 'en.xml' in the archive" AA raise tvdb_error(e) AA File "C:\SickChill\SickChill\lib\tvdb_api\tvdb_api.py", line 629, in _getetsrc AA epsEt = self._getetsrc(url, language=language) AA File "C:\SickChill\SickChill\lib\tvdb_api\tvdb_api.py", line 874, in _getShowData AA self._getShowData(key, self.config['language'], True) AA File "C:\SickChill\SickChill\lib\tvdb_api\tvdb_api.py", line 946, in __getitem__ AA s = t[self.indexer_id] AA File "C:\SickChill\SickChill\sickbeard\show_queue.py", line 324, in run AA Traceback (most recent call last): 2019-11-16 23:49:03 ERROR SHOWQUEUE-ADD :: [e094759] Unable to look up the show in C:\SickChill\The Mandalorian on theTVDB using ID 361753, not using the NFO. Delete .nfo and try adding manually again.: "There is no item named 'en.xml' in the archive" 2019-11-16 23:49:02 INFO SHOWQUEUE-ADD :: theTVDB: {u'useZip': True, u'apikey': u'F9C450E78D99172E', u'cache': u'C:\\SickChill\\Data\\cache\\indexers\\theTVDB', u'language': 'en'} 2019-11-16 23:49:02 INFO SHOWQUEUE-ADD :: Starting to add show by ShowDir: C:\SickChill\The Mandalorian AA None 2019-11-16 23:48:05 ERROR ThreadPoolExecutor-0_0 :: [e094759] Missing time zone for network: Apple TV+. Check valid network is set in indexer (theTVDB) before filing issue. AA None 2019-11-16 23:48:05 ERROR ThreadPoolExecutor-0_0 :: [e094759] Missing time zone for network: Peacock. Check valid network is set in indexer (theTVDB) before filing issue. 2019-11-16 23:39:16 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 23:39:16 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 23:29:19 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 23:19:31 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 23:19:22 INFO SEARCHQUEUE-DAILY-SEARCH :: Checking for scene exception updates for AniDB 2019-11-16 23:19:21 INFO SEARCHQUEUE-DAILY-SEARCH :: Checking for XEM scene exception updates for theTVDB 2019-11-16 23:19:21 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 23:19:20 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 23:19:20 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 22:39:32 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 22:39:20 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 22:39:20 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 22:39:20 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 22:39:16 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 22:39:16 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 22:29:18 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 21:59:32 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 21:59:24 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 21:59:20 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 21:59:20 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 21:39:16 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 21:39:16 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 21:29:18 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 21:19:30 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 21:19:22 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 21:19:19 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 21:19:19 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 20:39:29 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 20:39:21 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 20:39:19 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 20:39:19 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 20:39:15 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 20:39:15 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 20:29:17 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 20:29:14 INFO FINDPROPERS :: Completed the search for new propers, next check in approx. 3h, 59m 2019-11-16 20:29:14 INFO FINDPROPERS :: [OzNzb] :: Searching for any new PROPER releases from OzNzb 2019-11-16 20:29:14 INFO FINDPROPERS :: [NZBGeek] :: Searching for any new PROPER releases from NZBGeek 2019-11-16 20:29:14 INFO FINDPROPERS :: Beginning the search for new propers 2019-11-16 19:59:27 INFO SEARCHQUEUE-DAILY-SEARCH :: Error connecting to sab, no data returned 2019-11-16 19:59:27 INFO SEARCHQUEUE-DAILY-SEARCH :: Request failed: HTTPConnectionPool(host='192.168.1.100', port=8080): Max retries exceeded with url: /api?apikey=**********&name=https%3A%2F%2Fapi.nzbgeek.info%2Fapi%3Ft%3Dget%26id%3Da759ed6fc12c35cfaa99b0d103617daa%26apikey%3D9b4f90a886f84ec6c129951fa2ee8411&cat=tv&priority=1&mode=addurl&output=json (Caused by NewConnectionError(': Failed to establish a new connection: [Errno 10061] No connection could be made because the target machine actively refused it',)) () 2019-11-16 19:59:26 INFO SEARCHQUEUE-DAILY-SEARCH :: Sending NZB to SABnzbd 2019-11-16 19:59:26 INFO SEARCHQUEUE-DAILY-SEARCH :: Downloading See.S01E05.1080p.WEB-DL.DD5.1.H.264.1-CasStudio-Obfuscated from NZBGeek 2019-11-16 19:59:26 INFO SEARCHQUEUE-DAILY-SEARCH :: [NZBGeek] :: Quality of See.S01E05.1080p.WEB-DL.DD5.1.H.264.1-CasStudio-Obfuscated is 1080p WEB-DL 2019-11-16 19:59:26 INFO SEARCHQUEUE-DAILY-SEARCH :: [NZBGeek] :: Found result See.S01E05.1080p.WEB-DL.DD5.1.H.264.1-CasStudio-Obfuscated at https://api.nzbgeek.info/api?t=get&id=a759ed6fc12c35cfaa99b0d103617daa&apikey=********** 2019-11-16 19:59:20 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 19:59:19 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 19:59:19 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 19:39:15 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 19:39:15 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 19:29:17 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 19:19:27 INFO SEARCHQUEUE-DAILY-SEARCH :: Error connecting to sab, no data returned 2019-11-16 19:19:27 INFO SEARCHQUEUE-DAILY-SEARCH :: Request failed: HTTPConnectionPool(host='192.168.1.100', port=8080): Max retries exceeded with url: /api?apikey=**********&name=https%3A%2F%2Fapi.nzbgeek.info%2Fapi%3Ft%3Dget%26id%3Da759ed6fc12c35cfaa99b0d103617daa%26apikey%3D9b4f90a886f84ec6c129951fa2ee8411&cat=tv&priority=1&mode=addurl&output=json (Caused by NewConnectionError(': Failed to establish a new connection: [Errno 10061] No connection could be made because the target machine actively refused it',)) () 2019-11-16 19:19:26 INFO SEARCHQUEUE-DAILY-SEARCH :: Sending NZB to SABnzbd 2019-11-16 19:19:26 INFO SEARCHQUEUE-DAILY-SEARCH :: Downloading See.S01E05.1080p.WEB-DL.DD5.1.H.264.1-CasStudio-Obfuscated from NZBGeek 2019-11-16 19:19:26 INFO SEARCHQUEUE-DAILY-SEARCH :: [NZBGeek] :: Quality of See.S01E05.1080p.WEB-DL.DD5.1.H.264.1-CasStudio-Obfuscated is 1080p WEB-DL 2019-11-16 19:19:25 INFO SEARCHQUEUE-DAILY-SEARCH :: [NZBGeek] :: Found result See.S01E05.1080p.WEB-DL.DD5.1.H.264.1-CasStudio-Obfuscated at https://api.nzbgeek.info/api?t=get&id=a759ed6fc12c35cfaa99b0d103617daa&apikey=********** 2019-11-16 19:19:18 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 19:19:18 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 19:19:18 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 18:39:29 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 18:39:22 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 18:39:18 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 18:39:18 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 18:39:15 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 18:39:15 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 18:29:17 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 17:59:28 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 17:59:20 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 17:59:17 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 17:59:17 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 17:39:15 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 17:39:15 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 17:29:16 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 17:19:27 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 17:19:19 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 17:19:17 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 17:19:17 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 16:39:27 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 16:39:18 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 16:39:17 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 16:39:17 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 16:39:15 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 16:39:15 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 16:29:16 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 16:29:14 INFO FINDPROPERS :: Completed the search for new propers, next check in approx. 3h, 59m 2019-11-16 16:29:14 INFO FINDPROPERS :: Quality of See.S01E03.PROPER.1080p.HEVC.x265-MeGusta-Obfuscated is N/A 2019-11-16 16:29:14 INFO FINDPROPERS :: Quality of See.S01E04.PROPER.1080p.HEVC.x265-MeGusta-Obfuscated is N/A 2019-11-16 16:29:14 INFO FINDPROPERS :: Quality of See.S01E02.PROPER.1080p.HEVC.x265-MeGusta-Obfuscated is N/A 2019-11-16 16:29:14 INFO FINDPROPERS :: Quality of See.S01E05.PROPER.1080p.HEVC.x265-MeGusta-Obfuscated is N/A 2019-11-16 16:29:14 INFO FINDPROPERS :: [OzNzb] :: Searching for any new PROPER releases from OzNzb 2019-11-16 16:29:14 INFO FINDPROPERS :: [NZBGeek] :: Searching for any new PROPER releases from NZBGeek 2019-11-16 16:29:14 INFO FINDPROPERS :: Beginning the search for new propers 2019-11-16 15:59:29 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 15:59:21 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 15:59:16 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 15:59:16 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 15:39:15 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 15:39:15 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 15:29:16 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 15:19:27 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 15:19:20 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 15:19:16 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 15:19:16 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 14:39:26 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 14:39:19 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 14:39:16 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 14:39:16 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 14:39:15 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 14:39:15 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 14:29:15 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 13:59:25 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 13:59:17 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 13:59:15 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 13:59:15 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 13:39:15 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 13:39:15 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 13:29:15 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 13:19:23 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 13:19:16 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 13:19:15 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 13:19:15 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 12:39:20 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 12:39:19 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 12:39:15 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 12:39:14 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 12:39:14 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 12:39:14 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 12:29:15 INFO CHECKVERSION :: Request failed: HTTPConnectionPool(host='sickchill.github.io', port=80): Max retries exceeded with url: /sickchill-news/news.md (Caused by NewConnectionError(': Failed to establish a new connection: [Errno 11001] getaddrinfo failed',)) () 2019-11-16 12:29:15 WARNING CHECKVERSION :: Unable to contact github, can't check for update 2019-11-16 12:29:15 WARNING CHECKVERSION :: git fetch origin --prune returned : (128) fatal: unable to access 'https://github.com/SickChill/SickChill.git/': Couldn't resolve host 'github.com' 2019-11-16 12:29:14 WARNING MAIN :: Updating network timezones failed, this can happen from time to time. URL: http://sickchill.github.io/sb_network_timezones/network_timezones.txt 2019-11-16 12:29:14 INFO MAIN :: Request failed: HTTPConnectionPool(host='sickchill.github.io', port=80): Max retries exceeded with url: /sb_network_timezones/network_timezones.txt (Caused by NewConnectionError(': Failed to establish a new connection: [Errno 11001] getaddrinfo failed',)) () 2019-11-16 12:29:14 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 12:29:14 INFO TORNADO :: Starting SickChill on http://0.0.0.0:8081/ 2019-11-16 12:29:14 INFO MAIN :: Forcing web server to port 8081 2019-11-16 12:29:14 INFO MAIN :: Starting SickChill [master] using 'C:\SickChill\Data\config.ini' 2019-11-16 12:29:14 INFO MAIN :: 368747: Unable to find IMDb show info in the database 2019-11-16 12:20:56 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 12:20:47 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 12:20:43 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 12:20:43 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 12:00:35 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 12:00:35 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 11:50:42 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 11:40:56 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 11:40:46 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 11:40:42 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 11:40:42 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 11:00:45 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 11:00:45 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 11:00:42 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 11:00:42 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 11:00:35 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 11:00:35 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 10:57:11 INFO SEARCHQUEUE-BACKLOG-371446 :: No needed episodes found during backlog search for: [Heirs of the Night] 2019-11-16 10:57:08 INFO SEARCHQUEUE-BACKLOG-371446 :: [NZBGeek] :: Performing season pack search for Heirs of the Night 2019-11-16 10:56:44 INFO SEARCHQUEUE-BACKLOG-371446 :: [NZBGeek] :: Performing episode search for Heirs of the Night 2019-11-16 10:56:44 INFO SEARCHQUEUE-BACKLOG-371446 :: Beginning backlog search for: [Heirs of the Night] 2019-11-16 10:56:43 INFO SEARCHQUEUE-BACKLOG-361565 :: No needed episodes found during backlog search for: [See] 2019-11-16 10:56:43 INFO SEARCHQUEUE-BACKLOG-361565 :: [NZBGeek] :: Performing season pack search for See 2019-11-16 10:56:39 INFO SEARCHQUEUE-BACKLOG-361565 :: [NZBGeek] :: Performing episode search for See 2019-11-16 10:56:39 INFO SEARCHQUEUE-BACKLOG-361565 :: Beginning backlog search for: [See] 2019-11-16 10:56:35 INFO SEARCHQUEUE-BACKLOG-281534 :: No needed episodes found during backlog search for: [Killjoys] 2019-11-16 10:56:32 INFO SEARCHQUEUE-BACKLOG-281534 :: [NZBGeek] :: Performing season pack search for Killjoys 2019-11-16 10:55:59 INFO SEARCHQUEUE-BACKLOG-281534 :: [NZBGeek] :: Performing episode search for Killjoys 2019-11-16 10:55:59 INFO SEARCHQUEUE-BACKLOG-281534 :: Beginning backlog search for: [Killjoys] 2019-11-16 10:55:55 INFO SEARCHQUEUE-BACKLOG-320766 :: No needed episodes found during backlog search for: [The Good Fight] 2019-11-16 10:55:51 INFO SEARCHQUEUE-BACKLOG-320766 :: [NZBGeek] :: Performing season pack search for The Good Fight 2019-11-16 10:55:15 INFO SEARCHQUEUE-BACKLOG-320766 :: [NZBGeek] :: Performing episode search for The Good Fight 2019-11-16 10:55:14 INFO SEARCHQUEUE-BACKLOG-320766 :: Checking for scene exception updates from http://sickchill.github.io/scene_exceptions/scene_exceptions.json 2019-11-16 10:55:14 INFO SEARCHQUEUE-BACKLOG-320766 :: Beginning backlog search for: [The Good Fight] 2019-11-16 10:55:13 INFO SEARCHQUEUE-BACKLOG-317169 :: No needed episodes found during backlog search for: [Zapped!] 2019-11-16 10:55:10 INFO SEARCHQUEUE-BACKLOG-317169 :: [NZBGeek] :: Performing season pack search for Zapped! 2019-11-16 10:54:59 INFO SEARCHQUEUE-BACKLOG-317169 :: [NZBGeek] :: Performing episode search for Zapped! 2019-11-16 10:54:59 INFO SEARCHQUEUE-BACKLOG-317169 :: Beginning backlog search for: [Zapped!] 2019-11-16 10:54:58 INFO SEARCHQUEUE-BACKLOG-317169 :: No needed episodes found during backlog search for: [Zapped!] 2019-11-16 10:54:55 INFO SEARCHQUEUE-BACKLOG-317169 :: [NZBGeek] :: Performing season pack search for Zapped! 2019-11-16 10:54:39 INFO SEARCHQUEUE-BACKLOG-317169 :: [NZBGeek] :: Performing episode search for Zapped! 2019-11-16 10:54:39 INFO SEARCHQUEUE-BACKLOG-317169 :: Beginning backlog search for: [Zapped!] 2019-11-16 10:54:37 INFO SEARCHQUEUE-BACKLOG-317169 :: No needed episodes found during backlog search for: [Zapped!] 2019-11-16 10:54:34 INFO SEARCHQUEUE-BACKLOG-317169 :: [NZBGeek] :: Performing season pack search for Zapped! 2019-11-16 10:54:24 INFO SEARCHQUEUE-BACKLOG-317169 :: [NZBGeek] :: Performing episode search for Zapped! 2019-11-16 10:54:24 INFO SEARCHQUEUE-BACKLOG-317169 :: Beginning backlog search for: [Zapped!] 2019-11-16 10:54:24 INFO SEARCHQUEUE-BACKLOG-363642 :: No needed episodes found during backlog search for: [Sliced (2019)] 2019-11-16 10:54:21 INFO SEARCHQUEUE-BACKLOG-363642 :: [NZBGeek] :: Performing season pack search for Sliced (2019) 2019-11-16 10:54:14 INFO SEARCHQUEUE-BACKLOG-363642 :: [NZBGeek] :: Performing episode search for Sliced (2019) 2019-11-16 10:54:14 INFO SEARCHQUEUE-BACKLOG-363642 :: Beginning backlog search for: [Sliced (2019)] 2019-11-16 10:54:12 INFO SEARCHQUEUE-BACKLOG-300147 :: No needed episodes found during backlog search for: [SuperMansion] 2019-11-16 10:54:08 INFO SEARCHQUEUE-BACKLOG-300147 :: [NZBGeek] :: Performing season pack search for SuperMansion 2019-11-16 10:53:54 INFO SEARCHQUEUE-BACKLOG-300147 :: [NZBGeek] :: Performing episode search for SuperMansion 2019-11-16 10:53:54 INFO SEARCHQUEUE-BACKLOG-300147 :: Beginning backlog search for: [SuperMansion] 2019-11-16 10:53:53 INFO SEARCHQUEUE-BACKLOG-341663 :: No needed episodes found during backlog search for: [Titans (2018)] 2019-11-16 10:53:53 INFO SEARCHQUEUE-BACKLOG-341663 :: [NZBGeek] :: Performing season pack search for Titans (2018) 2019-11-16 10:53:49 INFO SEARCHQUEUE-BACKLOG-341663 :: [NZBGeek] :: Performing episode search for Titans (2018) 2019-11-16 10:53:49 INFO SEARCHQUEUE-BACKLOG-341663 :: Beginning backlog search for: [Titans (2018)] 2019-11-16 10:53:44 INFO SEARCHQUEUE-BACKLOG-253478 :: Sending NZB to SABnzbd 2019-11-16 10:53:44 INFO SEARCHQUEUE-BACKLOG-253478 :: Downloading American.Housewife.S04E07.720p.HDTV.x264-AVS-Obfuscated from NZBGeek 2019-11-16 10:53:44 INFO SEARCHQUEUE-BACKLOG-253478 :: [NZBGeek] :: Quality of American.Housewife.S04E07.720p.HDTV.x264-AVS-Obfuscated is 720p HDTV 2019-11-16 10:53:44 INFO SEARCHQUEUE-BACKLOG-253478 :: [NZBGeek] :: Found result American.Housewife.S04E07.720p.HDTV.x264-AVS-Obfuscated at https://api.nzbgeek.info/api?t=get&id=c87bd8ed05e9735cab00051699605039&apikey=********** 2019-11-16 10:53:44 INFO SEARCHQUEUE-BACKLOG-253478 :: [NZBGeek] :: Performing episode search for American Housewife 2019-11-16 10:53:44 INFO SEARCHQUEUE-BACKLOG-253478 :: Beginning backlog search for: [American Housewife] 2019-11-16 10:53:43 INFO SEARCHQUEUE-BACKLOG-77666 :: No needed episodes found during backlog search for: [The Amazing Race (US)] 2019-11-16 10:53:43 INFO SEARCHQUEUE-BACKLOG-77666 :: [NZBGeek] :: Performing season pack search for The Amazing Race (US) 2019-11-16 10:53:39 INFO SEARCHQUEUE-BACKLOG-77666 :: [NZBGeek] :: Performing episode search for The Amazing Race (US) 2019-11-16 10:53:39 INFO SEARCHQUEUE-BACKLOG-77666 :: Beginning backlog search for: [The Amazing Race (US)] 2019-11-16 10:53:38 INFO SEARCHQUEUE-BACKLOG-76733 :: No needed episodes found during backlog search for: [Survivor] 2019-11-16 10:53:38 INFO SEARCHQUEUE-BACKLOG-76733 :: [NZBGeek] :: Performing season pack search for Survivor 2019-11-16 10:53:34 INFO SEARCHQUEUE-BACKLOG-76733 :: [NZBGeek] :: Performing episode search for Survivor 2019-11-16 10:53:34 INFO SEARCHQUEUE-BACKLOG-76733 :: Beginning backlog search for: [Survivor] 2019-11-16 10:53:30 INFO SEARCHQUEUE-BACKLOG-275274 :: No needed episodes found during backlog search for: [Rick and Morty] 2019-11-16 10:53:26 INFO SEARCHQUEUE-BACKLOG-275274 :: [NZBGeek] :: Performing season pack search for Rick and Morty 2019-11-16 10:53:19 INFO SEARCHQUEUE-BACKLOG-275274 :: [NZBGeek] :: Performing episode search for Rick and Morty 2019-11-16 10:53:19 INFO SEARCHQUEUE-BACKLOG-275274 :: Beginning backlog search for: [Rick and Morty] 2019-11-16 10:53:13 INFO SEARCHQUEUE-BACKLOG-281618 :: Sending NZB to SABnzbd 2019-11-16 10:53:13 INFO SEARCHQUEUE-BACKLOG-281618 :: Downloading Fresh.Off.the.Boat.S06E07.720p.HDTV.x264-AVS-Obfuscated from NZBGeek 2019-11-16 10:53:13 INFO SEARCHQUEUE-BACKLOG-281618 :: [NZBGeek] :: Quality of Fresh.Off.the.Boat.S06E07.720p.HDTV.x264-AVS-Obfuscated is 720p HDTV 2019-11-16 10:53:09 INFO SEARCHQUEUE-BACKLOG-281618 :: [NZBGeek] :: Performing episode search for Fresh Off the Boat 2019-11-16 10:53:09 INFO SEARCHQUEUE-BACKLOG-281618 :: Beginning backlog search for: [Fresh Off the Boat] 2019-11-16 10:53:05 INFO SEARCHQUEUE-BACKLOG-110381 :: No needed episodes found during backlog search for: [Archer (2009)] 2019-11-16 10:53:05 INFO SEARCHQUEUE-BACKLOG-110381 :: [NZBGeek] :: Quality of Archer.2009.S10.720p.AMZN.WEB-DL.DDP5.1.H.264-NTb-Obfuscated is 720p WEB-DL 2019-11-16 10:53:00 INFO SEARCHQUEUE-BACKLOG-110381 :: [NZBGeek] :: Performing season pack search for Archer (2009) 2019-11-16 10:52:44 INFO SEARCHQUEUE-BACKLOG-110381 :: [NZBGeek] :: Performing episode search for Archer (2009) 2019-11-16 10:52:44 INFO SEARCHQUEUE-BACKLOG-110381 :: Beginning backlog search for: [Archer (2009)] 2019-11-16 10:52:41 INFO SEARCHQUEUE-BACKLOG-73141 :: No needed episodes found during backlog search for: [American Dad!] 2019-11-16 10:52:37 INFO SEARCHQUEUE-BACKLOG-73141 :: [NZBGeek] :: Performing season pack search for American Dad! 2019-11-16 10:52:05 INFO SEARCHQUEUE-BACKLOG-73141 :: [NZBGeek] :: Performing episode search for American Dad! 2019-11-16 10:51:54 INFO SEARCHQUEUE-BACKLOG-73141 :: Beginning backlog search for: [American Dad!] 2019-11-16 10:51:51 INFO SEARCHQUEUE-BACKLOG-73141 :: No needed episodes found during backlog search for: [American Dad!] 2019-11-16 10:51:47 INFO SEARCHQUEUE-BACKLOG-73141 :: [NZBGeek] :: Performing season pack search for American Dad! 2019-11-16 10:50:41 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 10:50:35 INFO FINDPROPERS :: Completed the search for new propers, next check in approx. 3h, 59m 2019-11-16 10:50:35 INFO FINDPROPERS :: [OzNzb] :: Searching for any new PROPER releases from OzNzb 2019-11-16 10:50:35 INFO FINDPROPERS :: [NZBGeek] :: Searching for any new PROPER releases from NZBGeek 2019-11-16 10:50:35 INFO FINDPROPERS :: Beginning the search for new propers 2019-11-16 10:50:34 INFO SEARCHQUEUE-BACKLOG-73141 :: [NZBGeek] :: Performing episode search for American Dad! 2019-11-16 10:50:34 INFO SEARCHQUEUE-BACKLOG-73141 :: Beginning backlog search for: [American Dad!] 2019-11-16 10:20:55 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 10:20:43 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 10:20:41 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 10:20:41 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 10:00:35 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 10:00:35 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 09:50:41 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 09:40:53 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 09:40:42 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 09:40:41 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 09:40:41 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 09:00:56 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 09:00:45 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 09:00:40 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 09:00:40 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 09:00:35 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 09:00:35 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 08:50:40 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 08:20:55 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 08:20:44 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 08:20:40 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 08:20:40 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 08:00:35 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 08:00:35 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 07:50:40 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 07:40:54 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 07:40:42 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 07:40:39 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 07:40:39 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 07:00:53 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 07:00:41 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 07:00:39 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 07:00:39 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 07:00:35 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 07:00:35 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 06:50:39 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 06:50:34 INFO FINDPROPERS :: Completed the search for new propers, next check in approx. 3h, 59m 2019-11-16 06:50:34 INFO FINDPROPERS :: [OzNzb] :: Searching for any new PROPER releases from OzNzb 2019-11-16 06:50:34 INFO FINDPROPERS :: [NZBGeek] :: Searching for any new PROPER releases from NZBGeek 2019-11-16 06:50:34 INFO FINDPROPERS :: Beginning the search for new propers 2019-11-16 06:20:52 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 06:20:39 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 06:20:39 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 06:20:39 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 06:00:35 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 06:00:35 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 05:50:39 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 05:40:54 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 05:40:43 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 05:40:38 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 05:40:38 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 05:00:52 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 05:00:41 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 05:00:38 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 05:00:38 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 05:00:34 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 05:00:34 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 04:50:38 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 04:50:37 INFO SHOWUPDATER :: Could not get the recently updated show data from theTVDB. Retrying later. Url was: http://thetvdb.com/api/Updates.php?type=series&time=1573702859 2019-11-16 04:50:37 INFO SHOWUPDATER :: Request failed: 403 Client Error: Forbidden for url: https://thetvdb.com/api/Updates.php?type=series&time=1573702859 () 2019-11-16 04:20:51 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 04:20:40 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 04:20:37 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 04:20:37 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 04:00:34 INFO FINDSUBTITLES :: No subtitles to download 2019-11-16 04:00:34 INFO FINDSUBTITLES :: Checking for missed subtitles 2019-11-16 03:50:38 INFO CHECKVERSION :: Checking for updates using GIT 2019-11-16 03:40:50 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 03:40:39 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes 2019-11-16 03:40:37 INFO DAILYSEARCHER :: No new released episodes found ... 2019-11-16 03:40:37 INFO DAILYSEARCHER :: Searching for new released episodes ... 2019-11-16 03:00:49 INFO SEARCHQUEUE-DAILY-SEARCH :: No needed episodes found 2019-11-16 03:00:37 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes
```

**Comment by manrata** [comment]
No metadata enabled.

**Comment by Matthewserta** [comment]
Pretty sure this is a widespread outage. Im having the same issue randomly. Nothing has changed in my setup. Also i dont think its getting existing shows either. are you sure yours is?

**Comment by darpified** [comment]
Definitely a prevalent issue, I'd say it's related to TVDB's recent changes/upgrades to the API.  Unable to add new shows, and it's not pulling existing shows.

**Comment by mmhere** [comment]
duplicate.  gated by fixes necessary on thetvdb.com side.  indeed a widespread outage.

[https://github.com/SickChill/SickChill/issues/5686](https://github.com/SickChill/SickChill/issues/5686)

**Comment by WebSpider** [comment]
Duplicate of #5686
